scripts/build.sh

```
@@ -13,36 +13,57 @@ DEBUG=false
+        ;;
     esac
 done
 
+echo "Starting build process..."
+
 if [ -z "${CAMINOBOT_PATH}" ]; then
-    # Camino-messenger-bot root folder
-    CAMINOBOT_PATH=$( cd "$( dirname "${BASH_SOURCE[0]}" )"; cd .. && pwd )
+    # camino-messenger-bot root folder
+    CAMINOBOT_PATH=$(
+        cd "$(dirname "${BASH_SOURCE[0]}")"
```

Shellcheck (0.10.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)